python/example_code/lookoutvision/datasets.py

```diff
@@ -0,0 +1,282 @@
+        param: manifest_s3_path:  The S3 location in which to store the created manifest file.
+        """
+
+        try:
+            output_manifest_file = "temp.manifest"
+
+            # Current date and time in manifest file format
+            #now=datetime.now()
+            dttm = datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%f")
+
+            # get bucket and folder from image and manifest file paths
+            bucket, prefix = image_s3_path.replace("s3://", "").split("/", 1)
+
+            manifest_bucket, manifest_prefix = manifest_s3_path.replace(
+                "s3://", ""
+            ).split("/", 1)
+
+            s3_client = boto3.client("s3")
+
+            # create local temp manifest file
+            with open(output_manifest_file, "w") as mfile:
+
+                logger.info("Creating manifest file")
+                # create JSON lines for anomalous images
+                response = s3_client.list_objects_v2(
+                    Bucket=bucket, Prefix=prefix + "anomaly/", Delimiter="/"
+                )
+                for file in response["Contents"]:
+                    image_path = "s3://{}/{}".format(bucket, file["Key"])
+                    manifest = Datasets.create_json_line(image_path, 1, dttm)
+                    mfile.write(json.dumps(manifest) + "\n")
+                # create json lines for normal images
+                response = s3_client.list_objects_v2(
+                    Bucket=bucket, Prefix=prefix + "normal/", Delimiter="/"
+                )
+                for file in response["Contents"]:
+                    image_path = "s3://{}/{}".format(bucket, file["Key"])
+                    manifest = Datasets.create_json_line(image_path, 0, dttm)
+                    mfile.write(json.dumps(manifest) + "\n")
+
+            # copy local manifest to target S3 location
+            logger.info("Uploading manifest file to %s", manifest_s3_path)
+            response = s3_client.upload_file(
+                output_manifest_file, manifest_bucket, manifest_prefix
+            )
+            # delete local manifest file
+            os.remove(output_manifest_file)
+
+        except ClientError as err:
+            print("S3 Service Error: {}".format(err))
+            raise
+
+        except Exception as err:
+            print(err)
+            raise
+        else:
+            logger.info("Completed manifest file creation and upload.")
+
+    @staticmethod
+    def create_json_line(image, label, dttm):
+        """
+        Creates a single JSON line for an image.
+        param: image: The S3 location for the image.
+        param: label: The label for the image (normal or anomaly)
+        param: dttm: The date and time that the JSON is created.
+        """
+
+        class_name = ""
+
+        if label == 0:
+            class_name = "normal"
+        elif label == 1:
+            class_name = "anomaly"
```

Comment:
More readable to use strings for this instead of numbers that transform to strings.
Or, if you're worried about bad inputs, use an Enum.